openfecli/tests/data/restructure_results_data.ipynb

```diff
@@ -12,6 +12,8 @@
     "import numpy as np\n",
     "import os \n",
     "import shutil\n",
+    "import openfe\n",
```

Comment:
I realize it's not good practice to have this notebook in here, but I'd like to keep it for data provenance until I revisit our tests for cleanup.